applications/nrf_desktop/doc/dfu.rst

```diff
@@ -18,115 +18,116 @@ Module events
 Configuration
 *************
 
-To perform the firmware upgrade, the bootloader must be enabled.
-More on how to enabled the bootloader can be found at :ref:`nrf_desktop_bootloader`.
+To perform the firmware upgrade, you must enable the bootloader.
+For more information on how to enable the bootloader, see the :ref:`nrf_desktop_bootloader` documentation.
 
-The DFU module is selected using ``CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_ENABLE``.
-Because it uses the :ref:`nrf_desktop_config_channel` for the transmission of the update image, it requires the transport option ``CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE`` to be selected.
+You can select the DFU module using the ``CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_ENABLE`` option.
+It requires the transport option ``CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE`` to be selected, as it uses :ref:`nrf_desktop_config_channel` for the transmission of the update image.
 
 When the update image is being received, it is stored in the dedicated partition on the flash memory.
-For more information about configuring the memory layout in the application, see :ref:`nrf_desktop_flash_memory_layout`.
+For more information on configuring the memory layout in the application, see the :ref:`nrf_desktop_flash_memory_layout` documentation.
 
 Implementation details
 **********************
 
-The DFU module implementation is centered around transmission and storage of the update image, and it can be broken down into the following stages:
+The DFU module implementation is centered around the transmission and the storage of the update image, and it can be broken down into the following stages:
 
 * `Protocol operations`_ - How the module exchanges information with the host.
 * `Partition preparation`_ - How the module prepares for receiving an image.
 
 The firmware transfer operation could also be carried out by :ref:`nrf_desktop_smp`.
-During the transfer operation, the ``dfu_lock`` (:file:`src/util/dfu_lock.h`) must be taken by the application module.
-The lock must also be taken for FLASH area erase operation.
-This is done to make sure that only one application module could modify secondary image FLASH partition.
-Modifying the data by multiple application modules would result in broken image that would be rejected by the bootloader.
+During the transfer operation, the ``dfu_lock``, located at :file:`src/util/dfu_lock.h`, must be taken by the application module.
```

Comment:
The mentioned header file describes the API, the lock is actually "located" in the `dfu_lock.c` file. Can we leave it with parenthesis ?